src/components/Modal/Modal.stories.jsx

```diff
@@ -6,19 +6,23 @@ export default {
   component: Modal,
 };
 
-const Template = (args) => {
+const Template = args => {
   const [visible, setVisible] = useState(args.visible);
   const onClose = () => setVisible(!visible);
+  const openModal = () => setVisible(true);
   return (
-    <Modal {...args} visible={visible} onClose={onClose}>
-      {args.children}
-    </Modal>
+    <div style={{ height: '500px' }}>
```

Comment:
Set height here to keep the iframe window on the Storybook page large enough to see the close button. Otherwise, since the modal isn't open at first, it stays at the minimum iframe height and the user can't see the close button so they can close the modal.